packages/rollup/src/rollup_bundle.bzl

```diff
@@ -0,0 +1,181 @@
+"Rules for running Rollup under Bazel"
+
+load("@build_bazel_rules_nodejs//internal/linker:link_node_modules.bzl", "module_mappings_aspect", "register_node_modules_linker")
+
+_ROLLUP_ATTRS = {
+    "srcs": attr.label_list(
+        doc = """JavaScript source files from the workspace.
+
+The file passed to entry_point is automatically added.
+""",
+        allow_files = [".js", ".json"],
```

Comment:
+".mjs"

Reply:
I think we'll have to give up on constraining this though - you could require() anything like svg or i18n translations